series/series_test.go

```diff
@@ -1778,3 +1778,43 @@ func TestSeries_Sum(t *testing.T) {
 		}
 	}
 }
+
+func TestSeries_Slice(t *testing.T) {
+	seriesWithErr := Ints([]int{})
+	seriesWithErr.Err = fmt.Errorf("slice index out of bounds")
+
+	tests := []struct {
+		j        int
+		k        int
+		series   Series
+		expected Series
+	}{
+		{
+			0,
+			3,
+			Ints([]int{1, 2, 3, 4, 5}),
+			Ints([]int{1, 2, 3}),
+		},
+		{
+			1,
+			1,
+			Ints([]int{1, 2, 3, 4, 5}),
+			Ints([]int{}),
+		},
+	}
```

Comment:
Nice to see good test cases! :-) 
Please add a case for `j < 0` and `k > s.Len()`, too.